flink-ml-lib/src/main/java/org/apache/flink/ml/feature/onehotencoder/OneHotEncoderParams.java

```diff
@@ -27,6 +27,8 @@
 /**
  * Params of OneHotEncoderModel.
  *
+ * <p>The `keep` option of {@link HasHandleInvalid} is not supported in {@link OneHotEncoderParams}.
```

Comment:
Yep. I added the comment here because we would support `SKIP` in the near future (I guess), but will never support `keep`.

Are you suggesting that we also mention that `SKIP` is not supported and remove it later in the near future?

Reply:
I think it could be better if we make the JavaDoc correspond to the actual behavior. We can modify the JavaDoc when we add support for `SKIP` in future. Thus it might be better to comment that `OneHotEncoder` only supports `ERROR` in its JavaDoc for now.

Reply:
Sounds good.